examples/enable-operator-and-auto-instrumentation/rendered_manifests/operator/admission-webhooks/operator-webhook.yaml

```diff
@@ -31,22 +31,22 @@ webhooks:
     failurePolicy: Fail
     name: minstrumentation.kb.io
     rules:
-    - apiGroups:
-        - opentelemetry.io
-      apiVersions:
-        - v1alpha1
-      operations:
-        - CREATE
-        - UPDATE
-      resources:
-        - instrumentations
-      scope: Namespaced
+      - apiGroups:
```

Comment:
Not blocking, but I’m concerned that our rendered version diverges from upstream. Redacted fields are fine, but the other changes seem like they should be upstreamed. Happy to review and approve that if we pursue it.

Reply:
The content is the same; just indentation has changed due to passing it through yq

Reply:
EKS add-ons enforce strict validation rules, and changes like this can sometimes cause issues. I won’t block this, but we may need to revisit it later, especially since the operator isn’t currently supported in the add-on, though we’re working to unblock that.

If possible, please try removing this pre-commit exclusion as part of this PR:
https://github.com/signalfx/splunk-otel-collector-chart/blob/fd4270019a7ad114ed2ca7ed5a78340f4e501ab8/.pre-commit-config.yaml#L26